src/components/Finish/FinishModal.tsx

```diff
@@ -34,6 +41,20 @@ export class FinishModal extends React.Component<FinishProps,FinishState> {
             name: playerName.substring(0,3).toUpperCase(),
         });
     };
+
+    sendStats() {
+        const params: RequestInit = {
+            method: "post",
+            headers: {"Content-Type":"application/json"},
+            body: JSON.stringify({
+                "name": this.state.name,
+                "time": this.props.speed,
```

Comment:
Should we rename this `time` field to `speed`?

Reply:
Yeah, but there's a bug card for this, so I figured we should change the backend there, not here